### PR TITLE
Wiring up

### PR DESCRIPTION
**Pull request by MichaelDJansen**
Wiring up PWA plugin

### CODE REVIEW

File: frontend/themes/app2/template.php [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <?php
 
 try {
+    $themeManager = new ThemeManager(new Theme());
+    $theme = $themeManager->getTheme();
 
     $curl_handle=curl_init();
-    curl_setopt($curl_handle, CURLOPT_URL, 'https://pwa-cdn.baobabsuite.com'.$_SERVER['REQUEST_URI']);
+    curl_setopt($curl_handle, CURLOPT_URL, $theme->getAppEndpoint().$_SERVER['REQUEST_URI']);
```
Comment (timbielawski) [inline frontend/themes/app2/template.php:8]: surely this is incorrect

**Review by timbielawski — APPROVED** [review]

File: admin/pages/whats-new.php [hunk 2]
```diff
@@ -1,222 +1,87 @@
...
+                                Initially all content will be activated to show on your mobile web application. Decide between various pages & categories to be activated/deactivated, by simply clicking on the row & selecting to activate/deactivate it. Categories & pages can also be rearranged by dragging the corresponding row to the desired position.
+                            </p>
                         </div>
                     </div>
                     <div class="spacer-0"></div>
-                    <?php 
-
-                     $step4 = get_home_url()."\wp-content\plugins\wordpress-pwa\admin\images\step4.png";   
 
+                    <?php
```
Comment (vjustinpietersen) [inline admin/pages/whats-new.php:67]: Not protesting or anything. I just wanted to know how this is more efficient than my way.

**Review by vjustinpietersen — APPROVED** [review]
Nice work.

**Review by dustanfranks — APPROVED** [review]
Doing all the right things, like you've been writing PHP for years.

**Pull request merged**